pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -172,20 +172,6 @@ buildPythonApplication rec {
 
   # skip failing tests due to darwin sandbox
   preCheck = lib.optionalString stdenv.hostPlatform.isDarwin ''
-    substituteInPlace kitty_tests/file_transmission.py \
-      --replace test_file_get dont_test_file_get \
-      --replace test_path_mapping_receive dont_test_path_mapping_receive \
-      --replace test_transfer_send dont_test_transfer_send
```

Comment:
This test seems flaky, so we should disable it for now.
https://github.com/NixOS/nixpkgs/pull/338070/checks?check_run_id=30805790618